actions/views/channel.js

```diff
@@ -69,6 +69,16 @@ export function switchToChannel(channel) {
     };
 }
 
+export function joinChannelById(channelId) {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const currentUserId = getCurrentUserId(state);
+        const currentTeamId = getCurrentTeamId(state);
+
+        return dispatch(joinChannel(currentUserId, currentTeamId, channelId));
```

Comment:
Out of curiosity, why is the team id required here? Seems like the channel would suffice in general?

Reply:
I can’t really remember but the redux action is expecting the teamId, so...